kolibri/core/tasks/job.py

```diff
@@ -185,15 +235,22 @@ def __init__(self, func, *args, **kwargs):
         if job_id is None:
             job_id = uuid.uuid4().hex
 
+        exception_type = kwargs.pop("exception_type", None)
+        if exception_type is None:
+            exception = None
+        else:
+            exception_message = kwargs.pop("exception_message", None)
+            exception = ErrorSavedWithJob(exception_message, exception_type)
+
```

Comment:
Following from the above recommendation regarding not having a separate exception class: 

These all should be removed.